src/model/openthaigpt_pretraining_model/lightning/utils.py

```diff
@@ -78,6 +83,10 @@ def seed_everything(seed):
     random.seed(seed)
 
 
+def compute_perplexity(loss):
```

Comment:
can you add type hint for loss

Reply:
in feat(model): Deepspeed for lightning fabric pull request #137